### DISCUSSION TITLE
Pre-requisites with Pythia Foundations

### DISCUSSION THREAD

**Discussion by acocac** [alan-turing-institute/environmental-ds-book#305] [Ideas]

**Comment by annefou** [comment]
## Pre-requisites with Project Pythia Foundations 

### Rationale

EDS Book notebooks are quite complex and use many different Python packages. To ease reuse by a large number of people, we suggest to add some prerequisites on top of each notebook and point to one or several training material from the [Project Pythia](https://projectpythia.org).


### Tasks

1. Select a notebook repository from the [EDS book Gallery](https://github.com/eds-book-gallery) and make the list of Python packages used in the main notebook file (`notebook.ipynb`).
2. Go to [Project Pythia](https://projectpythia.org/resource-gallery.html) and select the Pythia resources that are relevant for the selected EDS book notebook.
3. Create a PR that add a Pre-requisite section (see for example [Numpy basics](https://foundations.projectpythia.org/core/numpy/numpy-basics.html#prerequisites)),  in the notebook with links to the selected Pythia resources.
